ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -72,24 +72,37 @@
     {
         puzzleFilePath = @Model.Puzzle.PuzzleFile.Url.AbsoluteUri;
     }
+
     if (puzzleFilePath.Length > 0)
     {
-        if (puzzleFilePath.EndsWith(".pdf"))
-        {
-            embedType = 1;
-        }
-        else if ((puzzleFilePath.EndsWith(".htm")) || (puzzleFilePath.EndsWith(".html")))
+        if (!puzzleFilePath.ToLowerInvariant().StartsWith("https://puzzlehunt.azurewebsites.net"))
```

Comment:
This isn't right - should be:

```
string lowerPath = puzzleFilePath.ToLowerInvariant();
if (!lowerPath.StartsWith("https://puzzleserverteststore.blob.core.windows.net/") && !lowerPath.StartsWith("localhost"))
```

since the endpoint on puzzlehunt.azurewebsites.net is one of the ones we _want_.

Reply:
I did add the localhost check you mentioned here since that's also good to have in there